azure/Guidelines.md

```diff
@@ -1012,35 +1031,37 @@ All patterns that initiate a LRO either implicitly or explicitly create a [Statu
 Property | Type        | Required | Description
 -------- | ----------- | :------: | -----------
 `id`     | string      | true     | The unique id of the operation
-`kind`   | string enum | true     | The kind of operation
+`kind`   | string enum | true(*)  | The kind of operation
 `status` | string enum | true     | The operation's current status: "NotStarted", "Running", "Succeeded", "Failed", and "Canceled"
 `error`  | ErrorDetail |          | If `status`=="Failed", contains reason for failure
-`result` | object      |          | if Action LRO (POST or PUT) && `status`=="Succeeded", contains success result
+`result` | object      |          | If `status`=="Succeeded" && Action LRO (POST or PUT), contains success result if needed
 additional<br/>properties | | | Additional named or dynamic properties of the operation
 
-- Because services can support different kinds of operations, status monitor resources in this collection *must be* polymorphic; the `kind` property indicates the kind of long-running operation.
+(*): When a status monitor endpoint supports multiple operations with different result structures or additional properties,
+the status monitor *must be* polymorphic -- it **must** contain a `kind` property that indicates the kind of long-running operation.
```

Comment:
Nit: Why italic (one asterisk) in one place and bold (two asterisks) in another?